tests/build-utils/build-assert-e2e-img.sh

```diff
@@ -6,5 +6,23 @@ set -xe
 
 image_name=$1
 timestamp_file=$2
-docker build -t "$image_name" -f Dockerfile.asserts . $DOCKER_BUILD_OPTIONS
+
+if [ "$MULTI_ARCH_ASSERT_IMG" = false ]; then
+    docker build -t "$image_name" -f Dockerfile.asserts . $DOCKER_BUILD_OPTIONS
+else
+    # check if the image is in the container registry
+    manifest=$(docker manifest inspect "$image_name" 2>/dev/null || true)
+    if [ -z "$manifest" ]; then
+        echo "the e2e test asserts container image is not available in the remote registry. locally building and pushing into remote registry"
+        docker buildx build --push \
+            --progress=plain \
+            --platform ${PLATFORMS} \
+            --file Dockerfile.asserts \
+            --tag ${image_name} .
+    else
+        echo "the e2e test asserts container image is available in the remote registry. pulling into local environment"
```

Comment:
Why do you need to pull it? The cluster can get it from the registry

Reply:
in case if you enable `MULTI_ARCH_ASSERT_IMG` in a test environment where `KIND` used
https://github.com/jaegertracing/jaeger-operator/blob/53f3e0ac1bc6209b810c657c39c1fbda3a1aaf10/tests/build-utils/push-assert-e2e-img.sh#L13

Reply:
I'm not sure about this. If the images was pushed, it should be able to get it (with the vertx, it happens)